### PR TITLE
add planet name

### PR DESCRIPTION
**Pull request by jfiora**
image

### CODE REVIEW

File: src/components/Planet/PlanetInfo/PlanetDescription/PlanetName/PlanetName.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import './PlanetName.css';
+
+const PlanetName = ({name}) => {
+    return (
+        <h1 class="planetName">{name}</h1>
```
Comment (maruxan) [inline src/components/Planet/PlanetInfo/PlanetDescription/PlanetName/PlanetName.jsx]: Component class name should be PascalCase

File: src/components/Planet/PlanetInfo/PlanetDescription/PlanetName/PlanetName.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@import url('https://fonts.googleapis.com/css2?family=Antonio:wght@700&display=swap');
```
Comment (maruxan) [inline src/components/Planet/PlanetInfo/PlanetDescription/PlanetName/PlanetName.css]: Fonts must be imported globally via CDN in index.html

File: src/components/Planet/PlanetInfo/PlanetDescription/PlanetName/PlanetName.jsx (hunk 1 above)
Comment (maruxan) [reply src/components/Planet/PlanetInfo/PlanetDescription/PlanetName/PlanetName.jsx]: Same for [PR #4](https://github.com/jfiora/the-planets/pull/4)

File: src/components/Planet/PlanetInfo/PlanetDescription/PlanetName/PlanetName.css (hunk 2 above)
Comment (jfiora) [reply src/components/Planet/PlanetInfo/PlanetDescription/PlanetName/PlanetName.css]: created a new task in trello for that

File: src/components/Planet/PlanetInfo/PlanetDescription/PlanetName/PlanetName.jsx (hunk 1 above)
Comment (jfiora) [reply src/components/Planet/PlanetInfo/PlanetDescription/PlanetName/PlanetName.jsx]: done

**Pull request merged**